packages/h3/test/fixtures/01-basic/server.mjs

```diff
@@ -0,0 +1,5 @@
+import { H3, serve } from "h3";
+
+const app = new H3().get("/", () => "⚡️ Tadaa!");
+
+serve(app);
```

Comment:
We can/should do `export default app` (`serve` will use node.js APIs in compat mode)

Template: https://github.com/h3js/starter

(we have one more place to update)